packages/mobile/views/login/views/LegalUpdateView.svelte

```diff
@@ -0,0 +1,43 @@
+<script lang="ts">
+    import { Button, Checkbox, Text, TextType } from 'shared/components'
+    import { localize } from '@core/i18n'
+    import {
+        lastAcceptedTermsOfService,
+        lastAcceptedPrivacyPolicy,
+        TERMS_OF_SERVICE_VERSION,
+        PRIVACY_POLICY_VERSION,
+    } from '@core/app'
+    import { ConditionOfUse } from '../../../components'
```

Comment:
https://iotaledger.github.io/firefly/guides/coding-conventions/imports/#order
```suggestion
    import { Button, Checkbox, Text, TextType } from 'shared/components'
    import { ConditionOfUse } from '../../../components'
    import {
        lastAcceptedPrivacyPolicy,
        lastAcceptedTermsOfService,
        PRIVACY_POLICY_VERSION,
        TERMS_OF_SERVICE_VERSION,
    } from '@core/app'
    import { localize } from '@core/i18n'
```